_posts/2021-04-07-upgrade-improvements-and-migration.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title:  "Upgrade changes and API versions conversion"
+date: 2021-04-07
+author: jakub_stejskal
+---
+Upgrade process of Strimzi and Kafka is not trivial. 
+In Strimzi 0.22 we introduced several improvements, which will make the process easier for the users. 
+Moreover, we also introduced new API version `v1beta2` which will replace `v1beta1` because [Kubernetes 1.22 will remove support](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#customresourcedefinition-v122) for `apiextensions/v1beta1`.
+This blog post will show you how the upgrade process changed and what you should do to keep your cluster working without issues.
+
+<!--more-->
+
+### Upgrade improvements
+
+Legacy upgrade process was a little complicated.
+Users had to have fill Kafka `version`, but also `log.message.format.version` or `inter.broker.protocol.version` before the upgrade.
```

Comment:
We basically still recommend users to use this ... so I would not call it legacy. I would probably focus most here on the upgrading through each version and on the fact that you needed these even with operator hub etc.